src/main/java/uk/gov/mca/beacons/service/dto/RelationshipDTO.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.mca.beacons.service.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class RelationshipDTO {
+
+  private ArrayList<RelationshipData> data = new ArrayList<RelationshipData>();
```

Comment:
This type should be the interface `List` i.e: `private List<RelationshipData> data`